src/Umbraco.Web.UI/Umbraco.Web.UI.csproj

```diff
@@ -1973,6 +1973,8 @@
     <Content Include="Views\Partials\Grid\Editors\Textstring.cshtml" />
     <Content Include="Views\Partials\Grid\Bootstrap2.cshtml" />
     <Content Include="Views\Partials\Grid\Editors\Base.cshtml" />
+    <Content Include="Views\Partials\Grid\Bootstrap3-Fluid.cshtml" />

```

Comment:
Huh? 
![image](https://cloud.githubusercontent.com/assets/304656/20303382/65e38734-ab2b-11e6-9c1c-cb0687a5241a.png)

Anything we add we need to support and update all the time.. I'd prefer not to add stuff we don't need to.